lib/WebService/Async/CustomerIO/RateLimiter.pm

```diff
@@ -19,18 +19,20 @@ of request in time interval
 =head1 DESCRIPTION
 
 =cut
-sub configure {
-    my ($self, %args) = @_;
+
+
+sub _init {
+    my ($self, $args) = @_;
     for my $k (qw(limit interval)) {
-        die "Missing required argument: $k" unless exists $args{$k};
-        die "Invalid value for $k: $args{$k}" unless int($args{$k}) > 0;
-        $self->{$k} = delete $args{$k} if exists $args{$k};
+        die "Missing required argument: $k" unless exists $args->{$k};
+        die "Invalid value for $k: $args->{$k}" unless int($args->{$k}) > 0;
+        $self->{$k} = delete $args->{$k} if exists $args->{$k};
```

Comment:
this check is unnecessary due to the check on line 27: https://github.com/binary-com/perl-WebService-Async-CustomerIO/pull/2/files#diff-5f3b3d5265d1614c8deaecdf1e4a2ddcR27